paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -633,10 +633,10 @@ class Paparazzi @JvmOverloads constructor(
       "androidx.lifecycle.ViewTreeLifecycleOwner"
     )
     private val hasSavedStateRegistryOwnerRuntime = isPresentInClasspath(
-      "androidx.savedstate.SavedStateRegistryOwner"
+      "androidx.savedstate.ViewTreeSavedStateRegistryOwner"
     )
     private val hasAndroidxActivityRuntime = isPresentInClasspath(
-      "androidx.activity.OnBackPressedDispatcherOwner"
+      "androidx.activity.ViewTreeOnBackPressedDispatcherOwner"
```

Comment:
`View#setViewTreeOnBackPressedDispatcherOwner(owner)` is contained in `ViewTreeOnBackPressedDispatcherOwner` and was introduced in [androidx.activity.*:1.5.0](https://developer.android.com/jetpack/androidx/releases/activity#1.5.0).